server/views/pages/allocations/startAllocationsMigration.njk

```diff
@@ -63,6 +63,24 @@
                             errorMessage: errors | findError('courseActivityId')
                         }) }}
 
+                        {{ govukInput({
+                            label: {
+                                html: 'The date activities will start in DPS.'
+                            },
+                            id: "activityStartDate",
+                            name: "activityStartDate",
+                            classes: "govuk-input--width-20",
+                            hint: {
+                                text: "Example 2025-05-31, must be after today. Allocations active on this date will be migrated. Cannot be changed if defaulted from the Activities migration."
+                            },
+                            value: form.activityStartDate,
+                            disabled: form.protectStartDate,
+                            errorMessage: errors | findError('activityStartDate')
+                        }) }}
+                        {%if form.protectStartDate %}
+                          <input type="hidden" name="activityStartDate" data-qa="hidden-activityStartDate" value="{{ form.activityStartDate }}" />
+                        {% endif %}
```

Comment:
The main change is to pass the DPS activity start date through to the allocation migration so it can be used to exclude allocations that end in NOMIS before the start date. We protect the start date in this case - but leave it editable if the allocation migration doesn't hang of an activity migration.

Note that the hidden input field is because I couldn't work out how to make a `govukInput` readonly so ended up disabling it.